scylladb/sphinx-scylladb-theme
# Add Scylla Sphinx Theme 1.0

## Context

This PR adds the new design to the project ``scylladb/sphinx-scylladb-theme``. It moves the contents from the branch ``branch-1.0`` to ``master``.

## How to test this PR

1. Clone this PR. See [Checking out pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).
2. Run ``make preview`` to review the theme locally.
3. Accept the changes, or add comments.

## Showstoppers

- [x] The TOC on the left - when I click through several topics, what I want to see is: the topic I am on is blue &The parent of that topic (if relevant) is bold. **Waiting for design & confirmation**.
- [x] Tables - need many changes. I want to see a better design here. **Waiting for design & confirmation**.
- [x] The build takes too much time in scylla-docs.
- [x] Change admonition icons.
- [x] Headings - There is an issue w/ headings in that H3 is bigger than H2. I need to see a nice progressive hierarchy. At the lowest heading the text should be the same size as body text.
